platforms/arm.go

```diff
@@ -0,0 +1,69 @@
+package platforms
+
+import (
+	"bufio"
+	"log"
+	"os"
+	"runtime"
+	"strings"
+)
+
+// Structure to identify ARM platforms
+type PlatformArm struct {
```

Comment:
No, this cannot be a separate type.

Reply:
I will use an armCpuVariant string instead of  this separate type.